packages/amplify-codegen/src/commands/models.js

```diff
@@ -99,19 +101,48 @@ function loadSchema(apiResourcePath) {
   throw new Error('Could not load the schema');
 }
 
-function getModelOutputPath(context) {
+async function getModelOutputPath(context) {
   const projectConfig = context.amplify.getProjectConfig();
+  if(!projectConfig.frontend){
+    throw new Error('Frontend type is not configured.');
+  }
+
+  if(projectConfig.modelgen && projectConfig.modelgen.outputPath){
```

Comment:
In all places in this PR, formatting should include a space between `(` and `{` chars. e.g.,

```javascript
if (something.booleanValue) {
    // note the whitespace padding around if expression
}
```